src/lib.rs

```diff
@@ -66,7 +63,6 @@ extern crate hashbrown;
 #[cfg(feature = "serde")] #[macro_use] extern crate serde;
 #[cfg(all(test, feature = "serde"))] extern crate serde_json;
 #[cfg(all(test, feature = "serde"))] extern crate serde_test;
-#[cfg(all(test, feature = "serde"))] extern crate bincode;
 #[cfg(all(test, feature = "unstable"))] extern crate test;
```

Comment:
Why we removing just the line 69 here and not the rest?

Reply:
Hmm, I guess `cargo fix` did something weird here.